website/docs/theming/overview.mdx

```diff
@@ -34,92 +34,96 @@ The most common style modifiers are:
   schemes. For example, an outline button with a red color scheme.
 - **Color mode.** A component can change its visual styles based on color mode
   (light or dark).
+  
+## Theming local and global
 
-## Theming API
+- **Local.** Custom components can be themed locally using a `styled`-like API
+- **Global.** Both Chakra and Custom components can be themed via the central chakra theme using `themeKey`
+
+### Local Theming
 
 In styled components, you usually do this:
 
 ```js live=false
-const Button = styled("button")({
+const MyComponent = styled("div")({
   // any style you can imagine goes here
 })
 ```
```

Comment:
I change this example to not use `Button` because it's a bit confusing, since likely they'll just customize the existing out-of-the-box button instead of making a new one. I think it's more likely that they'll make a new component that is not out-of-the-box from Chakra, so I just use `MyComponent` to represent that.